opm/simulators/utils/PartiallySupportedFlowKeywords.cpp

```diff
@@ -107,7 +107,7 @@ partiallySupported()
          {
             "GRIDOPTS",
             {
-               {1,{false, allow_values<std::string> {"NO"}, "GRIDOPTS(TRANMULT): only the NO option is supported – value ignored"}}, // TRANMULT
+               {1,{false, allow_values<std::string> {"NO", "YES"}, "GRIDOPTS(TRANMULT): should be set to either NO or YES – will STOP"}}, // TRANMULT
```

Comment:
Either you need to remove the "will STOP" message, or make sure the simulator stops with "true".